plugin/pkg/shoot/oidc/clusteropenidconnectpreset/admission.go

```diff
@@ -128,6 +130,10 @@ func (c *ClusterOpenIDConnectPreset) Admit(_ context.Context, a admission.Attrib
 		return nil
 	}
 
```

Comment:
```suggestion

	// OIDCConfig is forbidden for clusters with kubernetes version >= 1.32.
	kubernetesVersion, err := semver.NewVersion(shoot.Spec.Kubernetes.Version)
	if err != nil {
		return apierrors.NewInternalError(fmt.Errorf("failed to parse shoot version: %w", err))
	}
	if versionutils.ConstraintK8sGreaterEqual132.Check(kubernetesVersion) {
		return nil
	}

```

We can do the check here. No need to error on clusters with version >= 1.32 as the deprecation of openID Connect config is documented here https://github.com/gardener/gardener/blob/v1.125.1/docs/usage/security/openidconnect-presets.md